custom_components/tuya_local/devices/Fresco_Hydrate_Ultra_Pet_Fountain.yaml

```diff
@@ -0,0 +1,111 @@
+name: Hydrate Ultra
+products:
+  - id: cuw2k3ho3e25fade
+    name: Hydrate Ultra
+
+primary_entity:
+  entity: light
+  icon: "mdi:water-pump"
+  dps:
+    - id: 101
+      type: boolean
+      name: switch
+    - id: 102
+      name: brightness
+      type: integer
+      range:
+        min: 1
+        max: 10
+
+secondary_entities:
+  - entity: button
+    name: Reset Filter Alarm
+    icon: "mdi:air-filter"
+    category: config
+    dps:
+      - id: 104
+        type: integer
+        name: button
+  - entity: sensor
+    name: Filter life
+    icon: "mdi:air-filter"
+    category: diagnostic
+    dps:
+      - id: 104
+        type: integer
+        name: sensor
+        precision: 0
+        range:
+          min: 0
+          max: 43200
+        mapping:
+          - scale: 1440
+            invert: true
+        unit: d
+  - entity: button
+    name: Reset Pump Alarm
+    icon: "mdi:pump"
+    category: config
+    dps:
+      - id: 105
+        type: integer
+        name: button
+  - entity: sensor
+    name: Pump maintenance
+    icon: "mdi:pump"
+    category: diagnostic
+    dps:
+      - id: 105
+        type: integer
+        name: sensor
+        precision: 0
+        range:
+          min: 0
+          max: 86400
+        mapping:
+          - scale: 1440
```

Comment:
Same here with class and scale

Reply:
Don't gonna do that, as it works correct. as well u have another supported fountain that has almost the same config, and there its the same way.